contrib/builder/rpm/generate.sh

```diff
@@ -96,5 +109,10 @@ for version in "${versions[@]}"; do
 
 	echo 'ENV AUTO_GOPATH 1' >> "$version/Dockerfile"
 
-	echo 'ENV DOCKER_BUILDTAGS selinux' >> "$version/Dockerfile"
+	DOCKER_BUILDTAGS="selinux"
+	if [[ "$distro" == "amazon" ]]; then
+		DOCKER_BUILDTAGS="$DOCKER_BUILDTAGS !journald"
```

Comment:
The Amazon Linux AMI doesn't use systemd yet and doesn't have journald libraries, but the CentOS 7 container has the journald development libraries installed if device-mapper-devel is installed.